todo_app/data/test_view_model.py

```diff
@@ -0,0 +1,63 @@
+from todo_app.data.item import Item
+from todo_app.data.view_model import ViewModel 
+import pytest
+
+# Arrange
+@pytest.fixture
+def do_items_list(): 
+    return [
+        Item(1, "My Todo Item 1", "To Do"),
```

Comment:
I don't see the benefit of adding 3 items in each list, your tests work just as well with only one, right?

Reply:
Fair point. I've reduced it to 1. Thanks.